### PR TITLE
gha: add release draft workflow

### PR DESCRIPTION
**Pull request by frealmyr**
Add Github Action for creating draft github releases, with auto-generated release notes based on pull requests between releases.

### CODE REVIEW

File: .github/workflows/release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Release Drafter
+
+on:
+  push:
```
Comment (jonasberdal) [inline .github/workflows/release-drafter.yml:4]: Når pusher vi rett på master?

File: .github/workflows/release-drafter.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    branches:
+      - master
```
Comment (jonasberdal) [inline .github/workflows/release-drafter.yml:6]: Når pusher vi rett på master?

Comment (frealmyr) [reply .github/workflows/release-drafter.yml:6]: Merging a pull request to master is considered a push as we create a new commit on the master branch.

File: .github/workflows/release-drafter.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: release-drafter/release-drafter@06d4616a80cd7c09ea3cf12214165ad6c1859e67
+        with:
+          config-name: release-drafter.yml
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (djfinnoy) [inline .github/workflows/release-drafter.yml:16]: ikke `secrets.GHA_PAT_REPO`?

**Review by jonasberdal — APPROVED** [review]

File: .github/workflows/release-drafter.yml (hunk 3 above)
Comment (frealmyr) [reply .github/workflows/release-drafter.yml:16]: We should use the integrated GHA token as this is a public repository 😉

**Review by djfinnoy — APPROVED** [review]

**Pull request merged**